opencga-catalog/src/main/java/org/opencb/opencga/catalog/migration/MigrationManager.java

```diff
@@ -431,7 +431,7 @@ private static Collection<URL> getUrls() {
         //  Currently they must contain "opencga" in the jar name.
         //  e.g.  acme-rockets-opencga-5.4.0.jar
         Collection<URL> urls = new LinkedList<>();
-        for (URL url : ClasspathHelper.forClassLoader()) {
+        for (URL url : ClasspathHelper.forPackage("org.opencb.opencga")) {
```

Comment:
Interesting change... was was the issue? would this be also needed for the ToolFactory?

Reply:
Actually yes. Since Java 9, I read that you need to define the package for which you want to get the classpath. Otherwise, it doesn't return anything. No migrations were being found because of that.